servant-server/src/Servant/Server.hs

```diff
@@ -235,7 +235,7 @@ hoistServer p = hoistServerWithContext p (Proxy :: Proxy '[])
 -- > │  └─ e/
 -- > │     └─•
 -- > ├─ b/
--- > │  └─ <capture>/
+-- > │  └─ <x::CaptureSingle>/
```

Comment:
Unrelated, but do you have any intuition on how hard it would be to add HTTP method information next to each leaf? :)

Reply:
I don't think it would be really hard ; it would look a lot like what I've done in #1556, except that you would have to add another argument to `StaticRouter`, and pass it in the `HasServer` instance for `Verb`. I expect the new router constructor to look like
```haskell
StaticRouter  (Map Text (Router' env a)) [(RouterEnv env -> a, HTTPMethod)]
```
Then the information would get passed along `routerStructure`, then to `routerLayout`

Reply:
Excellent, thank you :)